src/latexgen.cpp

```diff
@@ -315,7 +317,8 @@ static void writeLatexMakefile()
       << endl;
     t << "refman.pdf: refman.ps" << endl;
     t << "\tps2pdf refman.ps refman.pdf" << endl << endl;
-    t << "refman.dvi: clean refman.tex doxygen.sty" << endl
+    t << "refman.dvi: refman.tex doxygen.sty" << endl
+      << "\trm -f $(CLEAN_FILES)" << endl
```

Comment:
Why removing here the "clean" dependency and thus duplicating the code?

Reply:
See above.